contracts/actions/OperationRunner.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity >=0.7.6;
+pragma abicoder v2;
+
+import "../interfaces/mcd/IJoin.sol";
+import "../interfaces/mcd/IManager.sol";
+import "./ActionBase.sol";
+import "hardhat/console.sol";
+import "../ServiceRegistry.sol";
+
+struct Operation {
```

Comment:
in 0.8+ solc and in 0.7 with abie encoder v2 pragma constructs like:
```
pragma solidity ^0.7.6;
pragma abicoder v2;

struct Action{
    bytes32 id;
    bytes data;
}
struct Operation{
    Action[] actions;

}

contract Silly {
    function execute(Operation calldata op) public returns (bool){

    }
}

```

are possible